### ISSUE TITLE
It stays on the loading screen

### ISSUE THREAD

**Issue by Kelsier22** [Manuel-777/MTG-Arena-Tool#185]
I've uninstalled it several times with the v2.2.19, but it's still on the loading screen
I really enjoy this app

**Comment by Kelsier22** [comment]
![jjjjj](https://user-images.githubusercontent.com/47670497/52890098-0283d780-316a-11e9-9a03-10163796f708.PNG)

**Comment by lukash** [comment]
Same here. Installed from git. Tried checking out v2.2.18 to see if anything changes, same thing.

Console output:
```
> MTG-Arena-Tool@2.2.18 start /home/lu/mtga/MTG-Arena-Tool
> electron .

linux
APPIMAGE env is not defined, current application is not an AppImage
IPC LOG:  Renderer state: 1
IPC LOG:  SEND >> 0, get_database, V5zPWi, undefined
IPC LOG:  RECV << 0, get_database, V5zPWi, undefined
IPC LOG:  RECV << 0, get_database, {"65961":{"name":"Adanto Vanguard","set":"Ixalan","images":{"small":"\/small\/en\/xln\/1.jpg?1527427
IPC LOG:  SEND >> 1, get_status, MA1z2l, undefined
IPC LOG:  httpAsync: MA1z2l, 
IPC LOG:  RECV << 1, get_status, MA1z2l, undefined
IPC LOG:  RECV << 1, get_status, {
  "page": {
    "id": "hw05s0gzqdcs",
    "name": "MTG Arena",
    "url": "https://magicthegatheri
IPC LOG:  httpAsync: 

```

**Comment by LoganRaynes** [comment]
I've having this same issue. MTG Arena Tool also displays "output_log contains no player data".

I've tried deleting my output_log file but that didn't resolve it.

**Comment by Manuel-777** [comment]
Hello, we appear to have distinct issues here;
@Kelsier22 Please follow the [troubleshooting guide](https://github.com/Manuel-777/MTG-Arena-Tool/blob/master/TROUBLESHOOTING.md).

@lukash seems you are running from source? Try pulling latest version, or reinstalling all dependencies from scratch.

@LoganRaynes Update to v2.2.19

**Comment by lukash** [comment]
@Manuel-777 yes, from source.
```
$ git checkout master
...
$ git pull
...
$ git clean -xdf
Removing node_modules/
$ npm install

> electron@3.0.14 postinstall /home/lu/mtga/MTG-Arena-Tool/node_modules/electron
> node install.js


> MTG-Arena-Tool@2.2.19 postinstall /home/lu/mtga/MTG-Arena-Tool
> install-app-deps

  • please use as subcommand: electron-builder install-app-deps
  • electron-builder version=20.38.5
  • loaded configuration file=package.json ("build" field)
  • no native production dependencies
npm WARN MTG-Arena-Tool@2.2.19 No license field.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.7 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.7: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

added 785 packages from 522 contributors and audited 18807 packages in 10.258s
found 252 vulnerabilities (62 low, 189 moderate, 1 high)
  run `npm audit fix` to fix them, or `npm audit` for details
$ npm start

> MTG-Arena-Tool@2.2.19 start /home/lu/mtga/MTG-Arena-Tool
> electron .

linux
APPIMAGE env is not defined, current application is not an AppImage
IPC LOG:  Renderer state: 1
IPC LOG:  SEND >> 0, get_database, cm5nO9, undefined
IPC LOG:  RECV << 0, get_database, cm5nO9, undefined
IPC LOG:  RECV << 0, get_database, {"65961":{"name":"Adanto Vanguard","set":"Ixalan","images":{"small":"\/small\/en\/xln\/1.jpg?1527427
IPC LOG:  SEND >> 1, get_status, P1vLDx, undefined
IPC LOG:  httpAsync: P1vLDx, 
IPC LOG:  RECV << 1, get_status, P1vLDx, undefined
IPC LOG:  RECV << 1, get_status, {
  "page": {
    "id": "hw05s0gzqdcs",
    "name": "MTG Arena",
    "url": "https://magicthegatheri
IPC LOG:  httpAsync: 
```

It says "Downloading metadata", then "Metadata: OK" at the bottom, which disappears and then it's stuck at the Loading screen... I'm on Linux btw., can provide versions of installed software if you want.

From strace it appears to be waiting in possibly the main epoll_pwait on some IO, haven't figured more from it so far, not sure it's helpful.

**Comment by Manuel-777** [comment]
@lukash check [this issue](https://github.com/Manuel-777/MTG-Arena-Tool/issues/112) and let me know if that helps! Most linux issues are about finding the correct output_log location.

**Comment by lukash** [comment]
@Manuel-777 oh yes, that's exactly my problem. Thanks. Fixing it would be great... Like making sure you're reading a file and not a directory if that's what's happening? Weird it blocks like that anyway.

Also note the differences in the default path and what I've got (default install location), that's why I messed with the path in the first place.

```
tool: $WINEPREFIX/drive_c/user/lu/AppData/LocalLow/Wizards of the Coast/MTGA/output_log.txt
mine: $WINEPREFIX/drive_c/users/lu/AppData/LocalLow/Wizards Of The Coast/MTGA/output_log.txt
```
